### PR TITLE
MasterPackager Version 21.2.7837.0

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19669)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

**Comment by MasterPackager** [comment]
Hi, 
The hash seems to be correct, but we still get this error. 
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I'm still seeing this error:
image

**Comment by MasterPackager** [comment]
@ItzLevvie Could you provide some tips how to fix this? The hash is correct, so I don't know why we are getting this error.

**Comment by ItzLevvie** [comment]
> @ItzLevvie Could you provide some tips how to fix this? The hash is correct, so I don't know why we are getting this error.


I will try again later. It seems like it's downloading a 2 byte file.

**Comment by MasterPackager** [comment]
@ItzLevvie Our older manifests have the same problem. I guess its because they were updated to 1.0 schema.

File: manifests/m/MasterPackager/MasterPackager/21.2.7837.0/MasterPackager.MasterPackager.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- Application Packaging
+- Master Packager
+- MSI
+- Windows Installer
+ShortDescription: Master Packager is an application packaging tool to create and edit Microsoft Windows Installer (MSI) files and repackage other installation to MSI format.
+PackageUrl: https://www.masterpackager.com/
+Installers:
+- Architecture: x86
+  InstallerUrl: https://www.masterpackager.com/installer/professional/masterpackager_21.2.7837.0.msi
+  InstallerSha256: 81C9B08FCA3C3367A697FCF9BE174247702DA63A1473C4F96CE3C830255BCBA2
```
Comment (vedantmgoyal2009) [inline manifests/m/MasterPackager/MasterPackager/21.2.7837.0/MasterPackager.MasterPackager.yaml:18]: InstallerSha256: 81C9B08FCA3C3367A697FCF9BE174247702DA63A1473C4F96CE3C830255BCBA2

**Review by vedantmgoyal2009 — CHANGES_REQUESTED** [review]

**Review by vedantmgoyal9 — APPROVED** [review]

**Comment by MasterPackager** [comment]
@ItzLevvie Any luck finding out why its dowloading a 2 byte file?

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**